components/generic/ContentCard.vue

```diff
@@ -52,6 +52,17 @@
         >
           {{ $d(new Date(datetime), 'short') }}
         </time>
+        <template v-if="hitsText.length > 0">
+          <template v-for="(hit, index) in hitsText">
```

Comment:
this outside loop can be removed with the changes proposed for how hits are added to each item.
Also we only ever have one 'hit' so I think 'hitText' makes more sense as a variable.